src/ServiceControl.Monitoring/MetricReport.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.Metrics
+{
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// The reporting message.
+    /// </summary>
+    public class MetricReport : ICommand
```

Comment:
Probably "customizations" would be a better word. I mean that even though officially that could be anything, we still make an assumption this will be the exact type used, and if NSB.Metrics adds/modifies some data, then we'll break.